src/NLog.Web.AspNetCore/LayoutRenderers/AspNetTrackingConsentLayoutRenderer.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Http.Features;
+using NLog.Config;
+using NLog.LayoutRenderers;
+using NLog.Web.Enums;
+using NLog.Web.Internal;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Tracking Consent
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-tracking-consent:Property=CanTrack}
+    /// ${aspnet-tracking-consent:Property=HasConsent}
+    /// ${aspnet-tracking-consent:Property=IsConsentNeeded}
+    /// </remarks>
+    [LayoutRenderer("aspnet-tracking-consent")]
+    public class AspNetTrackingConsentLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
+        /// Specifies which of the 3 properties of ITrackingConsentFeature to emit
+        /// Defaults to CanTrack
+        /// </summary>
+        [DefaultParameter]
+        public TrackingConsentProperty Property { get; set; } = TrackingConsentProperty.CanTrack;
+
+        /// <summary>
+        /// Render Tracking Consent Information
```

Comment:
Can this be replaced by  `<inheritdoc/>` ?

Reply:
Complete